frontend/src/pages/Player/Toolbar/DevToolsWindowV2/DevToolsWindowV2.tsx

```diff
@@ -86,12 +86,13 @@ const DevToolsWindowV2: React.FC<
 		(level) => level !== LogSourceValue.All,
 	) as unknown as LogSource[]
 
-	const form = useFormState({
+	const formStore = useFormStore({
```

Comment:
type and status buttons are not working

![image](https://github.com/highlight/highlight/assets/1351531/f6cb7d18-a175-4899-8aba-38cf9e44dafd)

Reply:
Fixed in [b0cb293](https://github.com/highlight/highlight/pull/6417/commits/b0cb293f1c8667361c711d2219d4b4d04163994c) - good catch!

Reply:
still don't seem to be working  with b0cb293f1c8667361c711d2219d4b4d04163994c

Reply:
@Vadman97 looks like I fixed the placement issue with the popover, but there was another bug with triggering the popover consistently which I believe is fixed by b0b22f5bbb5fdbb1bc4aede3e393e86b17d97143. Sorry about that!